src/main/java/uk/gov/digital/ho/hocs/application/properties/LocalStackSqsProperties.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.digital.ho.hocs.application.properties;
```

Comment:
Not sure about these, why not just pass the parameters in? this falls into 'look busy' code IMO

Reply:
I tend to disagree, having them like this means you are explicit in the contract you expect. You can also ensure validation takes place, having just a `@Value` opens you up to mistakes..

Reply:
@ConfigurationProperties is a great way to encapsulate and validate related properties i.e. brings context